app/controllers/changesets_controller.rb

```diff
@@ -78,18 +78,35 @@ def feed
   # utility functions below.
   #------------------------------------------------------------
 
+  ##
+  # The API has few situations, where parameter is supposed
+  # to have boolean value (true/false), but previously their
+  # values were ignored and script checked only null/not null
+  # state.
+  # If string value is true, return boolean true, otherwize false.
+  def check_boolean(value)
```

Comment:
If this is needed by multiple controllers then it should be moved up to `application_controller.rb` instead of duplicating it.